addons/mail/static/tests/tours/mail_chatter_activity_test_tour.js

```diff
@@ -0,0 +1,49 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+/**
+ * This tour depends on data created by python test in charge of launching it.
+ * It is not intended to work when launched from interface. It is needed to test
+ * an action (action manager) which is not possible to test with QUnit.
+ * @see mail/tests/test_mail_chatter_activity.py
+ */
+registry.category("web_tour.tours").add("mail/static/tests/tours/mail_chatter_activity_test_tour.js", {
+    test: true,
+    steps: [
+        {
+            content: "Click on Activities",
```

Comment:
Not really, but just to indicate the thing we are doing right now for debugging. But indeed, it is pretty much redundant with the trigger, I can delete them

Reply:
I disagree, as generally triggers are unclear to most people. What is that button, where is it located, what are we doing ? Content purpose is to explain the test, so that we know what crashes.